src/app/core/services/guided-tour/guided-tour-items.service.ts

```diff
@@ -0,0 +1,392 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Injectable} from '@angular/core';
+import {GuidedTourItem} from '../../../shared/entity/guided-tour';
+import {Project} from '../../../shared/entity/project';
+import {Cluster, ClusterType, MasterVersion} from '../../../shared/entity/cluster';
+import {Node} from '../../../shared/entity/node';
+import {MachineDeployment} from '../../../shared/entity/machine-deployment';
+import {DigitaloceanSizes} from '../../../shared/entity/provider/digitalocean';
+import {Health} from '../../../shared/entity/health';
+
+@Injectable()
+export class GuidedTourItemsService {
+  getGuidedTourItems(): GuidedTourItem[] {
```

Comment:
Having all the data inside a single service will be a very high maintenance effort. Those items should be separate component-like objects, similar to dialogs, where we can define custom logic and view per item.